SQL/Archive/17.0/2016-08-19-IssueTracker.sql

```diff
@@ -0,0 +1,111 @@
+-- Add issues tab to the Loris Menu
+INSERT INTO `LorisMenu` (`Parent`, `Label`, `Link`, `Visible`, `OrderNumber`)
+VALUES (5, 'Issue Tracker', 'issue_tracker/', true, 8);
+
+
+-- Add user permissions
+-- Reporter
+INSERT INTO `permissions` (`code`, `description`, `categoryID`)
+VALUES ('issue_tracker_reporter', 'Can add a new issue, edit own issue, comment on all', 2);
+INSERT IGNORE INTO user_perm_rel (`userID`, `permID`) VALUES (
+  (SELECT ID FROM users WHERE UserID = 'admin'), (SELECT permID FROM permissions WHERE code = 'issue_tracker_reporter')
+);
+
+-- Developer
+INSERT INTO `permissions` (`code`, `description`, `categoryID`)
+VALUES ('issue_tracker_developer', 'Can re-assign issues, mark issues as closed, comment on all, edit issues.', 2);
+INSERT IGNORE INTO user_perm_rel (`userID`, `permID`) VALUES (
+  (SELECT ID FROM users WHERE UserID = 'admin'), (SELECT permID FROM permissions WHERE code = 'issue_tracker_developer')
+);
+
+-- LorisMenuPermissions
+INSERT INTO LorisMenuPermissions (MenuID, PermID)
+  SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='issue_tracker_reporter' AND m.Label='Issue Tracker';
+
+-- Issues Table
+CREATE TABLE `issues` (
```

Comment:
When running the patch, I get the following errors (most likely related to foreign key types):
```
[ERROR in query 7] Can't create table 'alexilea_sandbox.issues' (errno: 150)
[ERROR in query 9] Can't create table 'alexilea_sandbox.issues_history' (errno: 150)
[ERROR in query 10] Can't create table 'alexilea_sandbox.issues_comments' (errno: 150)
[ERROR in query 11] Can't create table 'alexilea_sandbox.issues_comments_history' (errno: 150)
```